about.md

```diff
@@ -152,42 +152,90 @@ tax adviser or the IRS about your particular tax situation.
-    inline; float: left; height: 4em; margin: 0 0.5em" /></a>
+### [Chan Zuckerberg Initiative][] (CZI)
+MDAnalysis has been supported by the [Essential Open Source for Science](https://chanzuckerberg.com/rfa/essential-open-source-software-for-science/) (EOSS) program from the CZI Donor-Advised Fund (DAF), an advised fund of Silicon Valley Community Foundation (funder DOI 10.13039/100014989)
 
+- EOSS5, 2022-253062 (**2022**): [MDAnalysis: Outreach and Project Manager](https://chanzuckerberg.com/eoss/proposals/mdanalysis-outreach-and-project-manager/) (**Personnel**: @IAlibay, @jennaswa, @micaela-matta, @orbeckst, @richardjgowers (*PI*))
+- EOSS4, DAF2021-237663, DOI [https://doi.org/10.37921/426590wiobus](https://doi.org/10.37921/426590wiobus) (**2021**): [MDAnalysis: Faster, Extensible Molecular Analysis for Reproducible Science](https://chanzuckerberg.com/eoss/proposals/mdanalysis-faster-extensible-molecular-analysis-for-reproducible-science/) (**Personnel**: @fiona-naughton, @hmacdope, @IAlibay, @ianmkenney, @lilyminium, @orbeckst (*PI*), @richardjgowers)
 
 <a href="https://chanzuckerberg.com/"><img
 	src="{{site.images}}/CZI_Logo.jpg" title="Chan Zuckerberg
 	Initiative" alt="Chan Zuckerberg Initiative" style="display:
 	inline; float: left; height: 4em; margin: 0 0.5em" /></a>
 
+### [Google](https://opensource.google/)
+The following contributors were sponsored to work on MDAnalysis through the [Google Summer of Code](https://summerofcode.withgoogle.com/) program.
+
+- **2023**: @marinegor, @xhgchen 
+- **2022**: @aya9aladdin, @BFedder
+- **2021**: @ojeda-e, @orioncohen
+- **2020**: @cbouy, @hmacdope, @yuxuanzhuang
+- **2019**: @NinadBhat
+- **2018**: @ayushsuhane, @davidercruz
+- **2017**: @utkbansal
+- **2016**: @fiona-naughton, @jdetle
+
+The following technical writers were sponsored to work on MDAnalysis through the [Google Season of Docs](https://developers.google.com/season-of-docs) program.
+
+- **2019**: @lilyminium
+
 <a href="https://summerofcode.withgoogle.com/"> <img
     src="https://developers.google.com/open-source/gsoc/images/gsoc2016-sun-373x373.png"
     title="Google Summer of Code" alt="Google Summer of Code"
     style="display: inline; float: left; height: 4em; margin: 0
     0.5em" /></a>
 
+<a href="https://developers.google.com/season-of-docs"> <img
+    src="https://developers.google.com/season-of-docs/images/SeasonofDocs_Icon_Grey_300ppi_trimmed.png"
+    title="Google Season of Docs" alt="Google Season of Docs"
+    style="display: inline; float: left; height: 4em; margin: 0 0.5em" /></a>
+
+### [National Science Foundation](https://www.nsf.gov/)
+Earlier work was partially supported by the NSF (as part of award ACI-1443054).
+
+- NSF DIBBS award, ACI-1443054 (**2014**): [CIF21 DIBBs: Middleware and High Performance Analytics Libraries for Scalable Data Science](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1443054) (**MDAnalysis Personnel**: @orbeckst)
```

Comment:
REU students 
- Ian Kenney @ianmkenney 
- Edis Jakupovic @edisj
- Henry Mull @hfmull
- Robert Delgado @rbrtdlgd
- Alia Lescoulie @Alescoulie
- Kathleen Clark @kaceyaurum

Reply:
Added - please ensure the years match up with the information you have.